module/proto/peggy/v1/attestation.proto

```diff
@@ -12,14 +10,24 @@ option go_package = "github.com/althea-net/peggy/module/x/peggy/types";
 // be handled
 enum ClaimType {
   option (gogoproto.goproto_enum_prefix) = false;
-
-  CLAIM_TYPE_UNKNOWN  = 0 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_UNKNOWN" ];
-  CLAIM_TYPE_DEPOSIT  = 1 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_DEPOSIT" ];
-  CLAIM_TYPE_WITHDRAW = 2 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_WITHDRAW"];
-  CLAIM_TYPE_ERC20_DEPLOYED = 3 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_ERC20_DEPLOYED"];
+  
+  CLAIM_TYPE_UNKNOWN             = 0 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_UNKNOWN" ];
```

Comment:
I think we should remove claim type unknown. I mean if we don't know what it is we should panic and or refuse the input.

Reply:
What is the default case? If we set 0 to CLAIM_TYPE_DEPOSIT then if the user does not set it we will assume its deposit. Setting Unknown is a recommended practice for protobuf and can help us in rejecting messages.

Reply:
oh wait this is a way we deal with go's really dumb and dangerous default behavior? I suppose that's acceptable

Reply:
Enums in proto should start at 0 in proto, 0 is generally used as the default value. It is also common practice in go.